### ISSUE TITLE
Convert any element to code block

### ISSUE THREAD

**Issue by stephanieo2** [soflyy/oxygen-bugs-and-features#570]
**Describe the feature you'd like to see included in Oxygen.**
I'd like to be able to convert any clement to a code block or edit it's source code while maintaining previously applied styles

**What are the use cases for this feature?**
ex.  convert an h1 to a code block to put a size span on some of the text.  Or nest a link onto some of the text

**Examples of this feature or functionality.**
Gutenburg does this

**Comment by albertso** [comment]
You can already insert spans and links in headings and texts with the top toolbar that appears in oxygen while editing the text.

If you really want to convert to a code block, you can inspect any element in frontend with chrome and copy the raw html and paste it in a code block.

You'll have to manually recreate/copy the styles in oxygen (in a stylesheet or with selectors), since the original styles are linked to the original element by id, and they'll disappear when you remove it.

Oxygen would have to do all this by itself, and it wouldn' work for elements that require javascript, or have other classes in the markup, because css won't be included in if the block isn't there.

Long story short, I think it's a bit complex for there to be a function to do this automatically.
